.travis.yml

```diff
@@ -21,6 +21,8 @@ env:
     - TEST_BLACKLIST="moveit_ros_perception"
     - CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unused-parameter -Wno-unused-but-set-parameter -Wno-unused-function"
     - WARNINGS_OK=false
+    - CCACHE_SLOPPINESS="include_file_ctime,include_file_mtime"
```

Comment:
This seems to be dangerous if ccache only looks at the include file's timestamps but not their contents.

Reply:
This has to do with what it decides to put in the ccache not what it compares with.

From the man page:

include_file_ctime
By default, ccache also will not cache a file if it includes a header whose ctime is too new. This option disables that check.

include_file_mtime
By default, ccache will not cache a file if it includes a header whose mtime is too new. This option disables that check.